datacontract/imports/dbt_importer.py

```diff
@@ -51,6 +51,23 @@ def read_dbt_manifest(manifest_path: str) -> Manifest:
     return manifest
 
 
+def _get_primary_keys(manifest: Manifest, node: ManifestNode):
+    node_unique_id = node.unique_id
+    if isinstance(node, ModelNode):
```

Comment:
Unfortunately, the `infer_primary_key` function only exists in the `ModelNode` class, so we have narrowed down the target to the `ModelNode` class only.